pkg/helm/handlers/handler_test.go

```diff
@@ -0,0 +1,178 @@
+package handlers
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/openshift/console/pkg/auth"
+	"github.com/openshift/console/pkg/helm/fake"
+	"helm.sh/helm/v3/pkg/release"
+)
+
+var fakeReleaseList = []*release.Release{
+	{
+		Name: "Test",
+	},
+}
+
+var fakeRelease = release.Release{
+	Name: "Test",
+}
+
+var fakeReleaseManifest = "manifest-data"
+
+func fakeHelmHandler() HelmHandlers {
+	return HelmHandlers{
+		getActionConfigurations: fake.GetFakeActionConfigurations,
+	}
+}
+
+func TestHelmHandlers_HandleHelmList(t *testing.T) {
```

Comment:
Test names should reflect the context under testing, `invalid test` and `valid test` do not tell much about it. Also, what is the benefit of prefixing test methods with `TestHelmHandlers_`?

I took a freedom to refactor this function, to demonstrate you how the readability could improved, please take a look here: https://github.com/akashshinde/console/compare/fix-odc-2871-helm-err-response...pedjak:fix-odc-2871-helm-err-response If you are ok with the approach, please apply it and apply the similar patter on the other test methods.